releases/release-announcement-guide.md

```diff
@@ -0,0 +1,306 @@
+
+
+# Purpose
+
+This document reviews the Kubeflow Community’s process for a blog post and a recorded presentation, which are delivered with each Kubeflow release.   The blog post and presentation are used to increase the Community’s understanding of new features in a Kubeflow release.  
+
+
+## Initial note
+
+Both of these deliveries (blog and presentation) have much of the same content.   The presentation can usually be completed in one shot with some quick edits.  It provides valuable information that can be used in the blog.  The blog requires more precision, and therefore takes multiple iterations.  It is suggested to start the blog before the presentation.
+
+
+# Blog Post
+
+
+## Blog Overview
+
+
+
+* Publish a blog of 1,000 words or less that reviews the highlights of new release
+* The blog will promote themes and the release’s top features.  The blog is a summary of the release, and more detailed information is provided in links to GitHub Issues, Pull Requests (PR), additional (Working Group or User) blogs, repos or other resources.
+* The post uses the GitHub PR fast pages process, which documented here, ​​https://github.com/kubeflow/blog#blog-submission-instructions
+    * This PR is an example of the 1.4 blog post, [https://github.com/kubeflow/blog/pull/109](https://github.com/kubeflow/blog/pull/109) 
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the blog’s creation and release process,  creates initial version, asks and incorporates edits, keeps process moving, provides updates to Community
+* GitHub specialist - 10 hours
+    * Creates PR, helps to check content, incorporate edits to PR
+* Content creators, editors and approvers - 2 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Final Approver - 4 hours
+    * Kubeflow Steering Committee lead
+
+
+## Example 
+
+An example of a release blog post is here: https://blog.kubeflow.org/kubeflow-1.4-release/
+
+Note - A sample of the work activities and timeline are detailed in a subsequent section of this document.
+
+
+# Presentation
+
+
+## Presentation overview
+
+30 minute slide presentation recorded and presented by the Kubeflow Community Product Management, the Release Team and the Working Group Leads.   The presentation should be posted on the [Kubeflow Youtube channel](https://www.youtube.com/channel/UCNSJwHkt2f_cP_UN2q-yJlw).
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the process, creates initial presentation outline, asks working groups for content and speakers, reviews and incorporates edits, keeps, provides updates to Community, keeps the process moving
+* Content creators, editors and approvers - 3 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Video Editor
+    * We need a pro at editing videos (that are currently created via Zoom)
```

Comment:
I think it might be sufficient to say someone with basic editing skills.

Reply:
ok